pages/[name]/territories.js

```diff
@@ -25,6 +27,7 @@ export default function UserTerritories ({ ssrData }) {
           query={USER_WITH_SUBS}
           variables={variables}
           destructureData={data => data.userSubs}
+          subActionDropdown={!!me}
```

Comment:
i wish there were a better way to do this. it's hard to track booleans dependency injected getting passed several components deep.

anyway, just wanted to note this for myself.

Reply:
I think I get what you're getting at. Moved the `useMe` hook directly into `TerritoryList`, seems to work fine that way without needing to track the dependency from here.